sphinx/themes/basic/static/basic.css_t

```diff
@@ -320,12 +320,17 @@ div.sidebar {
     background-color: #ffe;
     width: 40%;
     float: right;
+    clear: right
```

Comment:
Could you add a semicolon here?

Reply:
Oh, sorry, I somehow missed that ... I've added it in 57fe9ddc5aac53f51ba3eb25a661f3bdd49cb832.